frontend/src/metabase/query_builder/components/DataSelector.jsx

```diff
@@ -859,12 +859,14 @@ const TablePicker = ({
       },
     ];
     return (
-      <div>
+      <div style={{ width: 300, overflowY: "auto" }}>
         <AccordionList
           id="TablePicker"
           key="tablePicker"
           className="text-brand"
           sections={sections}
+          maxHeight={Infinity}
+          width={"100%"}
```

Comment:
Do we need these `width={"100%"}`? It seems to be working for me without, but I probably missed some cases.

Reply:
Yes, since `width:100%` is the full width without the scrollbar. That was why I called it hacky, since it's doesn't feel clean, but that's what you get when you mix CSS, JS and React I guess.

In Firefox, I specifically noticed it in Native variable sidebar, when it was showing more elements than what would fit in the Popover (like I showed in the screenshots).
If I remove the width, then it looks okay, but hovering the elements will show it's overflowing on the right, because of how scrollbar-width is calculated differently.

Let me do a bit more of testing, just so I'm totally sure about everything, since this is such a prominent function.